src/Microsoft.Metadata.Visualizer/ILVisualizer.cs

```diff
@@ -129,6 +129,8 @@ public LocalInfo(string name, object type, bool isPinned, bool isByRef)
         public virtual string VisualizeUserString(uint token) => $"0x{token:X8}";
         public virtual string VisualizeSymbol(uint token, OperandType operandType) => $"0x{token:X8}";
         public virtual string VisualizeLocalType(object type) => $"0x{type:X8}";
+        public virtual string VisualizeSingle(float single) => $"{single:G7}";
+        public virtual string VisualizeDouble(double @double) => $"{@double:G15}";
```

Comment:
Why not name the parameter `value`?

Reply:
I certainly could, this is just what made sense to me.